pkg/controller/policymetrics/metrics.go

```diff
@@ -11,12 +11,12 @@ import (
 var (
 	policyStatusGauge = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
-			Name: "ocm_policy_status",
+			Name: "governance_policy_status_info",
```

Comment:
I thought `policy_` should be first, and `_info` replaces `_status`. Then the insights metric was going to be `policy_report_info`